doc/IO.org

```diff
@@ -77,3 +78,37 @@ See [[../testing-data/house-votes-84.data][house-votes-84.data]] and [[../testin
 If the following lines have an entry more then the first line, the first entry
 will become the name of the object.  Otherwise, objects will be given arbitrary
 names.
+
+
+** FCA containing context, lattice and implication sets
```

Comment:
A promotional headline might be appropriate. Isn't the JSON format capable of storing a complete FCA data study?